### ISSUE TITLE
Trouble with some Brew code and converting to PDF

### ISSUE THREAD

**Issue by camanimal** [stolksdorf/homebrewery#218]
Hi there! I love the template site, and I have used it with success a few time, but my latest brew won't convert properly to pdf for some reason. It looks fine in the edit window but then turns into one clumped up page in the pdf window. Any thoughts? Many thanks for the cool site!

**Browser Type/Version**: Chrome Version 54.0.2840.71 (64-bit)

**Issue Description**: It's a 6 page brew that looks perfectly layed-out in the edit window, but in the pdf window it turns into a single page with jumbled images and text. 

**Brew code to reproduce**: <details><summary>Click to expand code</summary>
```

<div style='margin-top:100px'></div>

<img 
  src='https://s-media-cache-ak0.pinimg.com/736x/04/4a/cc/044acce6f8da483acc4663bed9ea4c9f.jpg' 
  style='width:550px; height:550px; position:absolute; right:125px' />

<style>
  .phb#p1{ text-align:center; }
  .phb#p1:after{ display:none; }
</style>


<div style='margin-top:1200px'></div>
# Ranger of the Beast Conclave

<div class='wide'>
##### Some rangers are so connected to the creatures of the wild that they consider one another kin. Rangers of the Beast Conclave develop an undying bond with a beast companion - their most trusted allies, their protectors, their soul mates.
<div class='pageNumber'>1</div>
<div class='footnote'>RANGER OF THE BEAST CONCLAVE</div>
\page

<img 
  src='http://i.imgur.com/24mGbSm.jpg' 
  style='width:480px; height:370px; position:absolute; right:160px' />
  <div style='margin-top:800px'></div>
<div class='classTable wide'>


##### Ranger of the Beast Conclave––––––––––––––––––––––––––––––––––Spell Slots per Level
| Level | Proficiency Bonus | Features | Spells Known | 1st | 2nd | 3rd | 4th | 5th |
|:---:|:---:|:---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| 1st | +2 | Favored Enemy, Favored Terrain | — | — | — | — | — | — | 
| 2nd | +2 | Fighting Style, Spellcasting | 2 | 2 | — | — | — | — |
| 3rd | +2 | Primeval Awareness, Beast Companion | 3 | 3 | — | — | — | — |
| 4th | +2 | Ability Score Improvement | 3 | 3 | — | — | — | — |
| 5th | +3 | Coordinated Attack, Additional Favored Terrain  | 4 | 4 | 2 | — | — | — | 
| 6th | +3 | Greater Favored Enemy | 4 | 4 | 2 | — | — | — |
| 7th | +3 | Beast Defense | 5 | 4 | 3 | — | — | — | 
| 8th | +3 | Ability Score Improvement, Fleet of Foot | 5 | 4 | 3 | — | — | — 
| 9th | +4 |Additional Favored Terrain | 6 | 4 | 3 | 2 | — | — |
| 10th | +4 | Natural Explorer, Hide in Plain Sight | 6 | 4 |3 | 2 | — | — |
| 11th | +4 | Storm of Fangs | 7 | 4 | 3 | 3 | — | — | 
| 12th | +4 | Ability Score Improvement | 7 | 4 | 3 | 3 | — | — |
| 13th | +5 | — | 8 | 4 | 3 | 3 | 1 | —
| 14th | +5 | Vanish| 8 | 4 | 3 | 3 | 1 | — |
| 15th | +5 | Superior Beast Defense | 9 | 4 | 3 | 3 | 2 | — |
| 16th | +5 | Ability Score Improvement | 9 | 4 | 3 | 3 | 2 | — |
| 17th | +6 | — | 10 | 4 | 3 | 3 | 3 | 1 |
| 18th | +6 | Feral Senses | 10 | 4 | 3 | 3 | 3 | 1 | 
| 19th | +6 | Ability Score Improvement | 11 | 4 | 3 | 3 | 3 | 2 
| 20th | +6 | Foe Slayer | 11 | 4 | 3 | 3 | 3 | 2 |
</div>


<div class='pageNumber'>2</div>

<div class='footnote'>RANGER OF THE BEAST CONCLAVE</div>

\page
## Class Features

As a Ranger, you gain the following class features
#### Hit Points
---
- **Hit Dice:** 1d10 per Ranger level
- **Hit Points at 1st Level:** 10 + your Constituion modifier
- **Hit Points at Higher Levels:** 1d10 (or 6) + your Constituion modifier per Ranger level after 1st
#### Proficiencies
---
- **Armor:** Light armor, medium armor, shields
- **Weapons:** Simple weapons, martial weapons
- **Tools:** None
---
- **Saving Throws:** Strength, Dexterity
- **Skills:** Choose three from Animal Handling,
  Athletics, Insight, Investigation, Nature,
  Perception, Stealth, and Survival
#### Equipment

You start with the following equipment, in addition to the equipment granted by your background:
- (a) scale mail or (b) leather armor
- (a) two shortswords or (b) two simple melee
  weapons
- (a) a dungeoneer’s pack or (b) an explorer’s pack
- A longbow and a quiver of 20 arrows
## Favored Enemy

Beginning at 1st level, you have significant
experience studying, tracking, hunting, and
interacting with beasts in the wild.
You gain an expertise in the Animal Handling skill. If you did not choose this skill, then you gain it, and if you chose it, then your proficiency bonus is doubled when rolling this skill check. Also, you receive a 
+2 bonus to damage rolls with weapon attacks
against all beasts,
you have advantage on Wisdom (Survival) checks
to track beasts, as well as on
Intelligence checks to recall information about
them.
---

You may also choose a single humanoid species to be your Favored Enemy (orcs, goblins, dwarves, kobolds, etc.) You gain the above bonuses against this species, and you also are proficient in their language.
## Favored Terrain

You are particularly familiar with one type of natural
environment and are adept at surviving in
such regions. Choose one type of favored terrain: **arctic,
coast, desert, forest, grassland, mountain, swamp,
or the Underdark**. You gain the following
benefits while traveling in this terrain:
---
- • Difficult terrain doesn’t slow your group’s
  travel.
- • Your group can’t become lost except by
  magical means.
- • Even when you are engaged in another activity
  while traveling (such as foraging, navigating,
  or tracking), you remain alert to danger.
- • If you are traveling alone, you can move
  stealthily at a normal pace.
- • On a successful Survival check while you tracking, you also learn general numbers and sizes of your quarry as well as how long since they passed through the area.
  ___

```
```

You also gain the following benefits during combat in your Favored Terrain:
---
- • You ignore difficult terrain.
- • You have advantage on initiative rolls.
- • On your first turn during combat, you have
  advantage on attack rolls against creatures
  that have not yet acted.
  ___
  You can choose additional favored terrain types at 5th
  and 9th level.
  ___
  ## Fighting Style
  At 2nd level, you adopt a particular style of
  fighting as your specialty. Choose one of the
  following options. You can’t take a Fighting Style
  option more than once, even if you later get to
  choose again.
  ___
  #### Archery
  You gain a +2 bonus to attack rolls you make
  with ranged weapons.
  ___
  #### Defense
  ___
  While you are wearing armor, you gain a +1
  bonus to AC.
  ___
#### Dueling

When you are wielding a melee weapon in one
hand and no other weapons, you gain a +2 bonus
to damage rolls with that weapon.
---
#### Two-Weapon Fighting

When you engage in two-weapon fighting, you
can add your ability modifier to the damage of
the second attack.
---
## Spellcasting

By the time you reach 2nd level, you have
learned to use the magical essence of nature to
cast spells, much as a druid does. See chapter 10
for the general rules of spellcasting and Page 6 for the Ranger Spell List.
---
#### Spell Slots

The Ranger table shows how many spell slots
you have to cast your spells of 1st level and
higher. To cast one of these spells, you must
expend a slot of the spell’s level or higher. You
regain all expended spell slots when you finish a
long rest.
For example, if you know the 1st-level spell
animal friendship and have a 1st-level and a 2ndlevel
spell slot available, you can cast animal
friendship using either slot.
---
#### Spells Known of 1st Level and Higher

You know two 1st-level spells of your choice
from the ranger spell list.
The Spells Known column of the Ranger table
shows when you learn more ranger spells of
your choice. Each of these spells must be of a
level for which you have spell slots. For instance,
when you reach 5th level in this class, you can
learn one new spell of 1st or 2nd level.
Additionally, when you gain a level in this class,
you can choose one of the ranger spells you
know and replace it with another spell from the
ranger spell list, which also must be of a level for
which you have spell slots.

<div class='pageNumber'>3</div>

<div class='footnote'>RANGER OF THE BEAST CONCLAVE</div>

\page

<div class='pageNumber'>4</div>

<div class='footnote'>RANGER OF THE BEAST CONCLAVE</div>
#### Spellcasting Ability

Wisdom is your spellcasting ability for your
Ranger spells, since your magic draws on your
attunement to nature. You use your Wisdom
whenever a spell refers to your spellcasting
ability. In addition, you use your Wisdom
modifier when setting the saving throw DC for a
ranger spell you cast and when making an attack
roll with one.
---

**Spell save DC = 8 +  proficiency bonus +
 WIS modifier**
---

**Spell attack modifier =  proficiency bonus +  WIS modifier**
---
## Primeval Awareness

Beginning at 3rd level, you can use your action and
expend one ranger spell slot to focus your awareness on
the region around you. For 1 minute per leveI of the spell
slot you expend, you can sense activity in any natural environment up to 1 mile radius around you. This feature uses limited real time communication between the Ranger and the DM, revealing general information in all directions or details in a specific direction, including what kind of creatures are nearby, their numbers, and/or their location.
---
## Beast Companion

At 3rd level, you learn to use your magic to
create a powerful bond with a creature of the
natural world.
---

By spending 24 hours in soltide on a Beast Bond Quest, you can call
forth an animal from the wilderness to serve as
your faithful companion. You can select
your companion from among the beasts native to the area you are in. You can only have one beast companion at a time.
---

If your beast companion is ever slain, the
magical link you share allows you to return it to
life. With another 24 hours spent in solitude in an environment native to your companion, you can summon another member of its species, channeling your companion’s spirit in order to inhabit a new body. You can return
a beast companion to life in this manner even
if you do not possess any part of its body.
---

If you use this ability to return a former 
companion to life while you are bonded to another beast, your current companion
leaves you and is replaced by the old
companion.
---
#### Beast Bond

Your beast companion incurs a variety of
changes while it is linked to you, both positive and negative.
- The beast loses its Multiattack
  action, if it has one.
- The companion obeys your commands as well as a trained animal possibly could. 
- It rolls for initiative like any other
  creature, but you determine its actions, decisions,
  attitudes, and so on. 
- If you are incapacitated or
  absent, your beast acts on its own (controlled by the DM.)
- 
  When using your Favored Terrain feature, you
  and your beast can both move
  stealthily at a normal pace.
  ___

```
```
- Your beast companion gains the benefits of
  your Favored Enemy feature, and of your
  Greater Favored Enemy feature when you gain
  that feature at 6th level. 
- Your beast companion has game
  statistics determined in part by your level. Your
  companion uses your proficiency bonus rather
  than its own. In addition to the areas where it
  normally uses its proficiency bonus, a beast
  companion also adds its proficiency bonus to its
  Armor Class,
  skills, saving throws, attack bonus, and damage rolls.
- 
  Your beast companion is
  proficient with all saving throws.
- For each level you gain after 3rd, your beast
  companion gains an additional hit die and
  increases its hit points accordingly.
- Whenever you gain the Ability Score
  Improvement class feature, your companion’s
  abilities also improve. Your companion can
  increase one ability score of your choice by 2, or
  it can increase two ability scores of your choice
  by 1. As normal, your companion can’t increase
  an ability score above 20 using this feature
  unless its description specifies otherwise.
- Your companion shares your alignment, and
  has a personality trait and a flaw that you can select or
  roll for below. Your
  beast shares your ideal, and its bond is
  always: **“The ranger who travels with me is my kin for whom I would gladly
  give my life.”**
  ___
  ##### BEAST TRAIT
  | D6 |–––TRAIT |
  |:----:|:-------------|
  | 1  | I’m dauntless in the face of adversity. |
  | 2  | Threaten my friends, threaten me. |
  | 3 | I stay on alert so others can rest. |
  | 4 | People see an animal and underestimate me. I use that to my advantage. |
  | 5 | I have a knack for showing up in the nick of time. |
  | 6 | I put my friends’ needs before my own in all things. |
  ___
  ##### BEAST FLAW
  | D6 |–––FLAW |
  |:----:|:-------------|
  | 1  | If there’s food left unattended, I’ll eat it. |
  | 2  | I growl at strangers, and all people except my ranger are strangers to me. |
  | 3 | Any time is a good time for a belly rub. |
  | 4 | I’m deathly afraid of water. |
  | 5 | My idea of hello is a flurry of licks to the face. |
  | 6 | I jump on you to say how much I love you. |
  ___
  ## Ability Score Improvement
  When you reach 4th level, and again at 8th, 12th,
  16th, and 19th level, you can increase one ability
  score of your choice by 2, or you can increase
  two ability scores of your choice by 1. As normal,
  you can’t increase an ability score above 20
  using this feature.
  \page

<div class='pageNumber'>5</div>

<div class='footnote'>RANGER OF THE BEAST CONCLAVE</div>

## Coordinated Attack

Beginning at 5th level, you and your beast
companion form a more potent fighting team.
When you use the Attack action on your turn, if
your companion can see you, it can use its
reaction to make a melee attack.
---
## Greater Favored Enemy

At 6th level, you are ready to hunt even deadlier
game. Choose a type of greater favored enemy:
**aberrations, celestials, constructs, dragons,
elementals, fey, fiends, giants, monstrosities, or the undead.** You learn your Greater Favored Enemy's language, you receive a 
+4 bonus to damage rolls with weapon attacks
against them,
you have advantage on Wisdom (Survival) checks
to track them and on
Intelligence checks to recall information about them.
---
## Beast Defense

At 7th level, while your companion can see you,
it has advantage on all saving throws.
---
## Fleet of Foot

Beginning at 8th level, you can use the Dash
action as a bonus action on your turn.
---
## Natural Explorer

At level 10, due to your time spent in the wild in all manners of terrain, you gain expertise in one of the three Ranger skills you chose, meaning that you double your proficiency bonus on any ability check rolls for that chosen skill.
---
## Hide in Plain Sight

Starting at 10th level, you can remain perfectly
still for long periods of time to set up ambushes.
When you attempt to hide on your turn, you
can opt to not move on that turn. If you avoid
moving, creatures that attempt to detect you
take a −10 penalty to their Wisdom (Perception)
checks until the start of your next turn. You lose
this benefit if you move or fall prone, either voluntarily or because of some external effect.
You are still automatically detected if any effect
or action causes you to no longer be hidden.
If you are still hidden on your next turn, you
can continue to remain motionless and gain this
benefit until you are detected.
---
## Storm of Fangs

At 11th level, your companion can use its action
to make a melee attack against each creature of its choice within 5 feet of it, with a separate
attack roll for each target.
---
## Vanish

Starting at 14th level, you can use the Hide
action as a bonus action on your turn. Also, you
can’t be tracked by nonmagical means, unless
you choose to leave a trail.

```
```
## Superior Beast Defense

At 15th level, whenever a target that your
companion can see hits it with an attack, it can
use its reaction to halve the attack’s damage
against it.
---
## Feral Senses

At 18th level, you gain preternatural senses that
help you fight creatures you can’t see. When you
attack a creature you can’t see, your inability to
see it doesn’t impose disadvantage on your
attack rolls.
You are also aware of the location of any
invisible creature within 30 feet of you, provided
 the creature isn’t hidden from you and you
aren’t blinded or deafened.
---
## Foe Slayer

At 20th level, you become an unparalleled
hunter. Once on each of your turns, you can add
your Wisdom modifier to the attack roll or the
damage roll of an attack you make against any of your Favored Enemies. You can
choose to use this feature before or after the roll,
but before any effects of the roll are applied.
\page

<div class='pageNumber'>6</div>

<div class='footnote'>RANGER OF THE BEAST CONCLAVE</div>

## RANGER SPELL LIST
---
### 1st level
- Alarm
- Animal Friendship
- Cure Wounds
- Detect Magic
- Detect Poison and Disease
- Ensnaring Strike
- Fog Cloud
- Goodberry
- Hail of Thorns
- Hunter's Mark
- Jump
- Longstrider
- Speak with Animals
### 2nd level
- Animal Messenger
- Barkskin
- Beast Sense
- Cordon of Arrows
- Darkvision
- Find Traps
- Lesser Restoration
- Locate Animals or Plants
- Locate Object
- Pass Without a Trace
- Protection from Poison
- Silence
- Spike Growth
### 3rd level
- Conjure Animals
- Conjure Barrage
- Daylight
- Lightning Arrow
- Nondetection
- Plant Growth
- Protection from Energy
- Speak with Plants
- Water Breathing
- Water Walk
- Wind Wall
### 4th level
- Conjure Woodland Beings
- Freedom of Movement
- Grasping Vine
- Locate Creature
- Stoneskin
### 5th Level
- Commune with Nature
- Conjure Volley
- Swift Quiver
- Tree Stride
```
</details>

**Comment by Tsurumah** [comment]
When you go to the Get PDF page, hit cancel on the print dialog, and then hit F12. That should bring up the html window. What you'll need to do is look for the pages in their respective `<div>` tags, which should look something like this:

![html](https://cloud.githubusercontent.com/assets/16673072/19837647/65431e5a-9e94-11e6-8fcc-40da2e91dc1c.jpg)

I've noticed that, at times, each page's div tag will be nested within the first page's div tag. What you'll have to do then is in that window drag each div tag to its proper place in order; only way I found to fix it when it happens.
